doc/source/train/user_guide.rst

```diff
@@ -346,20 +354,18 @@ Logging, Monitoring, and Callbacks
 ----------------------------------
 
 Ray Train has mechanisms to easily collect intermediate results from the training workers during the training run
-and also has a :ref:`Callback interface <train-callbacks>` to perform actions on these intermediate results (such as logging, aggregations, printing, etc.).
-You can use either the :ref:`built-in callbacks <train-builtin-callbacks>` that Ray Train provides,
-or implement a :ref:`custom callback <train-custom-callbacks>` for your use case.
+and also has a :ref:`Callback interface <train-callbacks>` to perform actions on these intermediate results (such as logging, aggregations, etc.).
+You can use either the :ref:`built-in callbacks <air-builtin-callbacks>` that Ray AIR provides,
+or implement a :ref:`custom callback <train-custom-callbacks>` for your use case. The callback API
+is shared with Ray Tune.
```

Comment:
Outside of the scope of this PR, but we should figure out how to cleanly separate the docs for Train/Tune/AIR. I think for a new user trying out Train the dependencies on Tune is a bit confusing.